cms/core/wagtail_hooks.py

```diff
@@ -35,3 +43,18 @@ def global_admin_css() -> str:
 
 register_snippet(ContactDetailsViewSet)
 register_snippet(GlossaryViewSet)
+
+
+@hooks.register("after_edit_page")
+def after_edit_page(request: "HttpRequest", page: "Page") -> None:
```

Comment:
note: we talked about this being `before_edit_page`, however this may be less noisy. One to get feedback on from DP